### PR TITLE
Update to 45

### PR DESCRIPTION
**Pull request by A6GibKm**

### CODE REVIEW

**Comment by A6GibKm** [comment]
bot, build

**Comment by A6GibKm** [comment]
Ready to merge

File: org.gnome.Characters.json [hunk 1]
```diff
@@ -54,7 +54,6 @@
                     "sha256": "42c773745d84ba14bc1cf1c4c6f4606148803a5cd337941c63964795f3c59d42",
```
Comment (bilelmoussaoui) [inline org.gnome.Characters.json:54]: There is a 45 release of this one, no?

Comment (bilelmoussaoui) [reply org.gnome.Characters.json:54]: also why the change to not important?

Comment (A6GibKm) [reply org.gnome.Characters.json:54]: It was set on this module (by me) by mistake, this way we get a single update when character releases. There is no 45 release.

**Pull request merged**